.npmrc

```diff
@@ -0,0 +1,2 @@
+auto-install-peers = true
```

Comment:
Have you tried?
```
engine-strict=true
```

Reply:
I didn't as it's not set on `material-ui`.
Do we agree that we want to be more strict about it? 🤔 
Is there enough incentive for it? What problems would it solve?
If I understand correctly, it would be mostly for `pnpm` version enforcing as having older `node` would basically fail somewhere else in the chain. 🙈 😆

Reply:
> Am I missing something?

Not much, it's enforcing the version of node.js and pnpm as specified by the engines field. In case of pnpm it indeed needs to be also added to engines. It makes sure nobody interacts with the repo with an outdated pnpm. Which is important to keep the behavior is consistent across collaborators and CI. Which avoids subtle bugs around e.g. the lock file.

Reply:
Added the entry to be consistent with toolpad. 👍